xrp-listing-guide.html

```diff
@@ -0,0 +1,778 @@
+<li class="level-3"><a href="#accounts">Accounts</a></li>
+<li class="level-3"><a href="#balance-sheets">Balance Sheets</a></li>
+<li class="level-2"><a href="#flow-of-funds">Flow of Funds</a></li>
+<li class="level-3"><a href="#deposit-xrp-into-exchange">Deposit XRP into Exchange</a></li>
+<li class="level-3"><a href="#trade-xrp-on-the-exchange">Trade XRP on The Exchange</a></li>
+<li class="level-3"><a href="#rebalance-xrp-holdings">Rebalance XRP Holdings</a></li>
+<li class="level-3"><a href="#withdraw-xrp-from-exchange">Withdraw XRP from Exchange</a></li>
+
+            </ul>
+        </div>
+    </div>
+      </aside>
+      <main class="main" role="main">
+    <div class='content'>
+        <h1 id="listing-xrp-as-an-exchange">Listing XRP as an Exchange</h1>
+<p>This document describes the steps that an exchange needs to take to list XRP. For details about other aspects of <code>rippled</code> and the Ripple Consensus Ledger (RCL), see the  <a href="https://ripple.com/build">Ripple Developer Center</a>.</p>
+<h2 id="alpha-exchange">Alpha Exchange</h2>
+<p>For illustrative purposes, this document uses a fictitious business called <em>Alpha Exchange</em> to explain the high-level steps required to list XRP. For the purposes of this document, Alpha Exchange:</p>
+<ul>
+<li>
+<p>Currently specializes in listing BTC/USD</p>
+</li>
+<li>
+<p>Wants to add BTC/XRP and XRP/USD trading pairs</p>
+</li>
+<li>
+<p>Maintains balances for all of its customers</p>
+</li>
+<li>
+<p>Maintains balances for each of its supported currencies</p>
+</li>
+</ul>
+<h3 id="user-benefits">User Benefits</h3>
+<p>Alpha Exchange wants to list BTC/XRP and XRP/USD trading pairs partially because listing these pairs will benefit its users. Specifically, this support will allow its users to:</p>
+<ul>
+<li>
+<p>Deposit XRP <em>to</em> Alpha Exchange <em>from</em> the RCL</p>
+</li>
+<li>
+<p>Withdraw XRP <em>from</em> Alpha Exchange <em>to</em> the RCL</p>
+</li>
+<li>
+<p>Trade XRP with other currencies, such as BTC, USD, amongst others</p>
+</li>
+</ul>
+<h2 id="prerequisites-for-supporting-xrp">Prerequisites for Supporting XRP</h2>
+<p>To support XRP, Alpha Exchange must:</p>
+<ul>
+<li>
+<p>Create and maintain new <a href="#accounts">accounts</a></p>
+</li>
+<li>
+<p>Create and maintain <a href="#balance-sheets">balance sheets</a></p>
+</li>
+</ul>
+<p>See also:</p>
+<ul>
+<li>
+<p><a href="https://ripple.com/build/gateway-guide/#gateway-compliance">Gateway Compliance</a> — Gateways and exchanges are different, but exchanges should also ensure that they are complying with local regulations and reporting to the appropriate agencies.</p>
+</li>
+<li>
+<p><a href="https://ripple.com/build/gateway-guide/#requirements-for-sending-to-rcl">Requirements for Sending to RCL</a></p>
+</li>
+<li>
+<p><a href="https://ripple.com/build/gateway-guide/#requirements-for-sending-to-rcl">Requirements for Sending to RCL</a></p>
```

Comment:
That's the generated HTML: it will be updated with my next commit to match the markdown source.